Pod/Native/NativeTestCase.swift

```diff
@@ -121,8 +121,8 @@ extension XCTestCase {
         }
         
         if let background = feature.background {
-            background.stepDescriptions.forEach(self.performStep)
+            background.stepDescriptions.forEach({ self.performStep($0) })
```

Comment:
Why these changes? Nothing wrong with them, just interested why the style change :)

Reply:
Ah, of course!